### PR TITLE
On non-VF nodes, use device serial to get the disk path

### PR DESCRIPTION
**Pull request by lujinda**

### CODE REVIEW

File: pkg/disk/bdf.go [hunk 1]
```diff
@@ -359,3 +360,37 @@ func forceDetachAllowed(disk *ecs.Disk, nodeID string) (allowed bool, err error)
 	// case 2
 	return inst.Status == InstanceStatusStopped, nil
 }
+
+var vfOnce = new(sync.Once)
+
+func IsVFNode() bool {
```
Comment (fredkan) [inline pkg/disk/bdf.go:366]: run this code when set GlobalConfigVar.DiskBdfEnable value is better. not need running for every check process.

Comment (lujinda) [reply pkg/disk/bdf.go:366]: I think  IsVFNode is more reliable.  The configuration(GlobalConfigVar.DiskBdfEnable) is easy to cause incorrect configuration due to human problems

**Pull request merged**